### PR TITLE
Remove plants with no utility from common sources

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
Checked each vendor/crate inventory against meal recipes and identified 2 with 0 use right now. This is mostly a small QOL for the kitchen.

**Changelog**

:cl: Rane
- remove: Removed plants with no current uses from botany vendors and crates.

### CODE REVIEW

**Review by EmoGarbage404 — APPROVED** [review]
This seems like a good change. I'm honestly surprised there aren't more plants than just these.

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/seeds.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@
     CornSeeds: 5
     EggplantSeeds: 5
     GalaxythistleSeeds: 5
-    LemonSeeds: 5
+#   LemonSeeds: 5 No recipes
     LingzhiSeeds: 5
     OatSeeds: 5
```
Comment (wrexbe) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/seeds.yml:19]: Missed one

**Review by wrexbe — CHANGES_REQUESTED** [review]

**Review by wrexbe — APPROVED** [review]

**Comment by wrexbe** [comment]
LemonSeeds exist on the Lighthouse map, and Oatseeds on Marathon map. Do you want to remove them?

**Comment by Peptide90** [comment]
I added oat seeds I’m sure as a way to get Oat Milk, for use in vegan / lactose free recipes. Then realised reagents don’t use tags so I can’t exchange things out in a recipe. Oh well.

**Pull request merged**